rskj-core/src/test/java/co/rsk/peg/BridgeTest.java

```diff
@@ -451,25 +442,25 @@ private BtcTransaction createTransaction() {
 
     @Test
     public void getGasForDataFreeTx() {
```

Comment:
This should `new` a new instance

Reply:
Changed to use @Before instead of static config :+1: